app/controllers/movies_controller.rb

```diff
@@ -8,4 +8,12 @@ def search
       @search = @movie_service.search(params[:q])
     end
   end
+
+  def detail
+    @user = current_user
+    @movie = @movie_service.movie_detail_for(params[:id])
+    @genres = @movie_service.genres(params[:id])
+    @casts = @movie_service.casts(params[:id])
+    @reviews = @movie_service.reviews(params[:id])
```

Comment:
What if we created a class for all of these objects? (Mainly for the purpose of DRYing out the controller

Reply:
yess!! I think we can do this when we refactor!